src/Apps/Order/Components/ShippingAddress.tsx

```diff
@@ -18,13 +20,20 @@ export const ShippingAddress = ({
   city,
   postalCode,
   region,
+  country,
 }: ShippingAddressProps) => {
-  const cityLine = city + (postalCode ? ` ${postalCode}` : "")
   return (
-    <Serif size="3t" style={{ whiteSpace: "pre-wrap" }}>
-      {[name, addressLine1, addressLine2, cityLine, region]
-        .filter(Boolean)
-        .join("\n")}
-    </Serif>
+    <>
+      <Serif size="3t">{name}</Serif>
+      <Serif size="3t">
+        {[addressLine1, (addressLine2 || "").trim()].filter(Boolean).join(", ")}
```

Comment:
Hmm should these two lines be joined with a comma? I would expect them to be on separate lines in the view.

Reply:
Maybe it's only stantard in the U.S, but this feels more natural to me. Also, [the mockup also uses a comma](https://zpl.io/bzWx4KE).